docs/readme_es.md

```diff
@@ -156,13 +156,13 @@ Puedes personalizar el aspecto de tu `Tarjeta de Estadísticas` o `Tarjeta de Re
 - `icon_color` - Color de icono si esta disponible _(hex color)_
 - `bg_color` - Color de fondo _(hex color)_
 - `hide_border` - Oculta el borde de la tarjeta _(booleano)_
-- `theme` - Nombre del tema, elige uno de [todos los temas disponible ](../themes/README.md)
+- `theme` - Nombre del tema, elige uno de [todos los temas disponible](../themes/README.md)
```

Comment:
This sentence is in plural. I think the correct way would be `todos los temas disponibles`